cvelib/cve_api.py

```diff
@@ -87,6 +90,14 @@ def post(self, path, **kwargs):
     def put(self, path, **kwargs):
         return self.http_request("put", path, **kwargs)
 
+    def create(self, cve_id, cve_json):
+        """Create a CVE from a JSON object representing the CNA container data.
+        Only the "affected", "descriptions", "providerMetadata", and "references" properties may be submitted.
+        See https://github.com/CVEProject/cve-services/issues/584 for details."""
+        response = self.post(f"/cve/{cve_id}/cna", json=cve_json)
```

Comment:
You can start the path without the leading slash and not have to do the stripping later. Or is there a specific reason to have it in there?

Reply:
Nope, good catch! I wondered why I was the first person to hit this behavior...